etna/lib/etna/command.rb

```diff
@@ -203,6 +203,7 @@ def subcommands
       @subcommands ||= self.class.constants.sort.reduce({}) do |acc, n|
         acc.tap do
           c = self.class.const_get(n)
+          next unless c.respond_to?(:instance_methods)
```

Comment:
Unrelated bug I found in the etna gem, due to using a const that wasn't another command.